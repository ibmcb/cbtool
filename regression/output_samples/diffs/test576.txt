|ai_31     |nullworkload   |1_x_tinyvm                                        |62.186.114.202  |1374527487     |aidrs_1                                 |31D48616-B02B-5245-AFD3-7958573C3CB0
|ai_32     |nullworkload   |1_x_tinyvm                                        |177.139.66.62   |1374527496     |aidrs_1                                 |5AF90A3B-C4FF-5B25-8E7A-66079F4F95E4
|vm_120        |tinyvm              |pico32      |62.186.114.202  |simhostb1                  |SUT                 |ai_31   |aidrs_1    |B5C2B723-FF25-57ED-9866-17D442719B50
|vm_121        |tinyvm              |pico32      |177.139.66.62   |simhostb4                  |SUT                 |ai_32   |aidrs_1    |81AA9B38-C8C7-5DB6-8B27-0AB52B9D66CA
|Used Memory                                          |44.32M                        
|Uptime (in seconds)                                  |13566279                      
|Total Connections Received                           |6495141                       
|Total Commands Processed                             |94136002                      
|Number of Keys                                       |683                           
|Storage Size                                         |16114180080.0                 
|Data Size                                            |13823593944.0                 
|Index Size                                           |550481904                     
|Average Object Size                                  |6600.31806217                 
|Collections                                          |60                            
|D30F1B96-9225-51C9-95FF-3355AD659B11 (simzone_d) VM RESERVATIONS|0                             
|3C091D65-A8D9-5ECF-9C95-36A39D5CAA39 (simzone_a) VM RESERVATIONS|0                             
|671172C3-2E80-5782-AB0C-C348A586106A (simzone_c) VM RESERVATIONS|0                             
|96342978-982B-5D60-9FF3-0C0B3F9C41A9 (simzone_b) VM RESERVATIONS|2                             
|9A5383E7-2829-576E-9AAC-B47A428B43D2 (aidrs_1) AI RESERVATIONS|2                             
