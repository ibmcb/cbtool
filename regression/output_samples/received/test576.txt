Going to wait until the value on counter "AI ARRIVED" is equal to 30 (currently it is equal to 29) waiting 5.0 seconds between samples. The counter is assumed to be increasing.
 status: Counter "AI ARRIVED" equals 29 after 5 seconds (the counter is assumed to be increasing).
 status: Counter "AI ARRIVED" equals 30 after 10 seconds (the counter is assumed to be increasing).
Waited 10 seconds until "AIs ARRIVED" was equal to 30.
The following AIDRSs are attached to this experiment (Cloud TESTCLOUD) :
|name                |pattern                |type              
|aidrs_1             |simplenw               |nullworkload      

The following AIs are attached to this experiment (Cloud TESTCLOUD) :
|name      |type           |sut                                               |cloud_ip        |arrival        |aidrs                                   |uuid
|ai_31     |nullworkload   |1_x_tinyvm                                        |62.186.114.202  |1374527487     |aidrs_1                                 |31D48616-B02B-5245-AFD3-7958573C3CB0
|ai_32     |nullworkload   |1_x_tinyvm                                        |177.139.66.62   |1374527496     |aidrs_1                                 |5AF90A3B-C4FF-5B25-8E7A-66079F4F95E4

The following VMs are attached to this experiment (Cloud TESTCLOUD) :
|name          |role                |size        |cloud_ip        |host_name                  |vmc_pool            |ai      |aidrs      |uuid
|vm_120        |tinyvm              |pico32      |62.186.114.202  |simhostb1                  |SUT                 |ai_31   |aidrs_1    |B5C2B723-FF25-57ED-9866-17D442719B50
|vm_121        |tinyvm              |pico32      |177.139.66.62   |simhostb4                  |SUT                 |ai_32   |aidrs_1    |81AA9B38-C8C7-5DB6-8B27-0AB52B9D66CA

The following statistics are available on this experiment (Cloud TESTCLOUD) :
------------------------- OBJECT STORE -----------------------
|Metric                                               |Value                         
|Used Memory                                          |44.32M                        
|Redis Version                                        |2.5.12                        
|Uptime (in seconds)                                  |13566279                      
|Total Connections Received                           |6495141                       
|Total Commands Processed                             |94136002                      
|Number of Keys                                       |683                           
------------------------- METRIC STORE -----------------------
|Metric                                               |Value                         
|MongoDB Version                                      |2.0.6                         
|Storage Size                                         |16114180080.0                 
|Data Size                                            |13823593944.0                 
|Index Size                                           |550481904                     
|Average Object Size                                  |6600.31806217                 
|Collections                                          |60                            
--------------------- EXPERIMENT OBJECTS ---------------------
|Object                                               |Count                         
|VMCs                                                 |4                             
|HOSTs                                                |20                            
|VMs                                                  |2                             
|SVMs                                                 |0                             
|AIs                                                  |2                             
|AIDRSs                                               |1                             
|VMCRSs                                               |0                             
|FIRSs                                                |0                             
------------------ EXPERIMENT-WIDE COUNTERS ------------------
|Counter                                              |Value                         
|VMC RESERVATIONS                                     |4                             
|VMCs ARRIVED                                         |16                            
|VMCs ARRIVING                                        |0                             
|VMCs DEPARTED                                        |12                            
|VMCs DEPARTING                                       |0                             
|VMCs FAILED                                          |0                             
|HOST RESERVATIONS                                    |0                             
|HOSTs ARRIVED                                        |0                             
|HOSTs ARRIVING                                       |0                             
|HOSTs DEPARTED                                       |0                             
|HOSTs DEPARTING                                      |0                             
|HOSTs FAILED                                         |0                             
|VM RESERVATIONS                                      |2                             
|VMs ARRIVED                                          |117                           
|VMs ARRIVING                                         |0                             
|VMs CAPTURING                                        |0                             
|VMs DEPARTED                                         |115                           
|VMs DEPARTING                                        |0                             
|VMs FAILED                                           |4                             
|SVM RESERVATIONS                                     |0                             
|SVMs ARRIVED                                         |0                             
|SVMs ARRIVING                                        |0                             
|SVMs DEPARTED                                        |0                             
|SVMs DEPARTING                                       |0                             
|SVMs FAILED                                          |0                             
|AI RESERVATIONS                                      |2                             
|AIs ARRIVED                                          |30                            
|AIs ARRIVING                                         |0                             
|AIs CAPTURING                                        |0                             
|AIs DEPARTED                                         |28                            
|AIs DEPARTING                                        |0                             
|AIs FAILED                                           |2                             
|AIDRS RESERVATIONS                                   |1                             
|AIDRSs ARRIVED                                       |1                             
|AIDRSs ARRIVING                                      |0                             
|AIDRSs DEPARTED                                      |0                             
|AIDRSs DEPARTING                                     |0                             
|AIDRSs FAILED                                        |0                             
|VMCRS RESERVATIONS                                   |0                             
|VMCRSs ARRIVED                                       |0                             
|VMCRSs ARRIVING                                      |0                             
|VMCRSs DEPARTED                                      |0                             
|VMCRSs DEPARTING                                     |0                             
|VMCRSs FAILED                                        |0                             
|FIRS RESERVATIONS                                    |0                             
|FIRSs ARRIVED                                        |0                             
|FIRSs ARRIVING                                       |0                             
|FIRSs DEPARTED                                       |0                             
|FIRSs DEPARTING                                      |0                             
|FIRSs FAILED                                         |0                             
|EXPERIMENT COUNTER                                   |385                           

 ---------------- VMC-WIDE COUNTERS ----------------
|D30F1B96-9225-51C9-95FF-3355AD659B11 (simzone_d) VM RESERVATIONS|0                             
|3C091D65-A8D9-5ECF-9C95-36A39D5CAA39 (simzone_a) VM RESERVATIONS|0                             
|671172C3-2E80-5782-AB0C-C348A586106A (simzone_c) VM RESERVATIONS|0                             
|96342978-982B-5D60-9FF3-0C0B3F9C41A9 (simzone_b) VM RESERVATIONS|2                             

 ---------------- AIDRS-WIDE COUNTERS ----------------
|9A5383E7-2829-576E-9AAC-B47A428B43D2 (aidrs_1) AI RESERVATIONS|2                             

