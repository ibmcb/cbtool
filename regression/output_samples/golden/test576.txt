Going to wait until the value on counter "AI ARRIVED" is equal to 30 (currently it is equal to 29) waiting 5.0 seconds between samples. The counter is assumed to be increasing.
 status: Counter "AI ARRIVED" equals 29 after 5 seconds (the counter is assumed to be increasing).
 status: Counter "AI ARRIVED" equals 30 after 10 seconds (the counter is assumed to be increasing).
Waited 10 seconds until "AIs ARRIVED" was equal to 30.
The following AIDRSs are attached to this experiment (Cloud TESTCLOUD) :
|name                |pattern                |type              
|aidrs_1             |simplenw               |nullworkload      

The following AIs are attached to this experiment (Cloud TESTCLOUD) :
|name      |type           |sut                                               |cloud_ip        |arrival        |aidrs                                   |uuid
|ai_31     |nullworkload   |1_x_tinyvm                                        |68.204.84.85    |1372702646     |aidrs_1                                 |336C8994-1ACE-516F-9AA7-6B89D1CC5933
|ai_32     |nullworkload   |1_x_tinyvm                                        |105.105.138.69  |1372702655     |aidrs_1                                 |E10ACE9D-73FB-52EB-939C-E7AA49DD5D5A

The following VMs are attached to this experiment (Cloud TESTCLOUD) :
|name          |role                |size        |cloud_ip        |host_name                  |vmc_pool            |ai      |aidrs      |uuid
|vm_120        |tinyvm              |pico32      |68.204.84.85    |simhosta0                  |SUT                 |ai_31   |aidrs_1    |F05058DA-4DCE-5C3F-8FBE-03E5FE7CC012
|vm_121        |tinyvm              |pico32      |105.105.138.69  |simhosta0                  |SUT                 |ai_32   |aidrs_1    |A8EEDBF4-016F-550E-9C56-3B48AAA04739

The following statistics are available on this experiment (Cloud TESTCLOUD) :
------------------------- OBJECT STORE -----------------------
|Metric                                               |Value                         
|Used Memory                                          |44.28M                        
|Redis Version                                        |2.5.12                        
|Uptime (in seconds)                                  |11741438                      
|Total Connections Received                           |6492952                       
|Total Commands Processed                             |93069028                      
|Number of Keys                                       |682                           
------------------------- METRIC STORE -----------------------
|Metric                                               |Value                         
|MongoDB Version                                      |2.0.6                         
|Storage Size                                         |13967917056.0                 
|Data Size                                            |12135654604.0                 
|Index Size                                           |503093808                     
|Average Object Size                                  |6343.91445927                 
|Collections                                          |62                            
--------------------- EXPERIMENT OBJECTS ---------------------
|Object                                               |Count                         
|VMCs                                                 |4                             
|HOSTs                                                |20                            
|VMs                                                  |2                             
|SVMs                                                 |0                             
|AIs                                                  |2                             
|AIDRSs                                               |1                             
|VMCRSs                                               |0                             
|FIRSs                                                |0                             
------------------ EXPERIMENT-WIDE COUNTERS ------------------
|Counter                                              |Value                         
|VMC RESERVATIONS                                     |4                             
|VMCs ARRIVED                                         |16                            
|VMCs ARRIVING                                        |0                             
|VMCs DEPARTED                                        |12                            
|VMCs DEPARTING                                       |0                             
|VMCs FAILED                                          |0                             
|HOST RESERVATIONS                                    |0                             
|HOSTs ARRIVED                                        |0                             
|HOSTs ARRIVING                                       |0                             
|HOSTs DEPARTED                                       |0                             
|HOSTs DEPARTING                                      |0                             
|HOSTs FAILED                                         |0                             
|VM RESERVATIONS                                      |2                             
|VMs ARRIVED                                          |117                           
|VMs ARRIVING                                         |0                             
|VMs CAPTURING                                        |0                             
|VMs DEPARTED                                         |115                           
|VMs DEPARTING                                        |0                             
|VMs FAILED                                           |4                             
|SVM RESERVATIONS                                     |0                             
|SVMs ARRIVED                                         |0                             
|SVMs ARRIVING                                        |0                             
|SVMs DEPARTED                                        |0                             
|SVMs DEPARTING                                       |0                             
|SVMs FAILED                                          |0                             
|AI RESERVATIONS                                      |2                             
|AIs ARRIVED                                          |30                            
|AIs ARRIVING                                         |0                             
|AIs CAPTURING                                        |0                             
|AIs DEPARTED                                         |28                            
|AIs DEPARTING                                        |0                             
|AIs FAILED                                           |2                             
|AIDRS RESERVATIONS                                   |1                             
|AIDRSs ARRIVED                                       |1                             
|AIDRSs ARRIVING                                      |0                             
|AIDRSs DEPARTED                                      |0                             
|AIDRSs DEPARTING                                     |0                             
|AIDRSs FAILED                                        |0                             
|VMCRS RESERVATIONS                                   |0                             
|VMCRSs ARRIVED                                       |0                             
|VMCRSs ARRIVING                                      |0                             
|VMCRSs DEPARTED                                      |0                             
|VMCRSs DEPARTING                                     |0                             
|VMCRSs FAILED                                        |0                             
|FIRS RESERVATIONS                                    |0                             
|FIRSs ARRIVED                                        |0                             
|FIRSs ARRIVING                                       |0                             
|FIRSs DEPARTED                                       |0                             
|FIRSs DEPARTING                                      |0                             
|FIRSs FAILED                                         |0                             
|EXPERIMENT COUNTER                                   |385                           

 ---------------- VMC-WIDE COUNTERS ----------------
|35310FEB-1E59-53F1-A37C-0E4387DFF893 (simzone_a) VM RESERVATIONS|2                             
|DAEF899C-8C95-5B76-BE3F-58349C261E66 (simzone_d) VM RESERVATIONS|0                             
|8E18B916-CC95-566E-887E-3032BA7F835A (simzone_c) VM RESERVATIONS|0                             
|3076C295-3106-5ADC-8C71-D21083902D7A (simzone_b) VM RESERVATIONS|0                             

 ---------------- AIDRS-WIDE COUNTERS ----------------
|033779F0-1E69-5DEA-8D15-A82CDEFAEF7C (aidrs_1) AI RESERVATIONS|2                             

